### DISCUSSION TITLE
1 Şubattan Sonra tekrar bir airdrop olacakmı?

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#1205] [Soru Cevap - Türkçe]
Yani şuan İbax Hakkında pek bir bilgimiz yok.  1 Şubattan sonra puanımıza göre airdrop alıcağımızı biliyoruz ama çok ama çok az dağıtıyorlar merak ettiğim konu işte bu acaba sonradan 2 etap gibi airdrop olacakmı bilen varsa yazzarsa çok sevinirim? 

![148614-black_and_white_robot_holding_umbrella_illustration-3840x2160](https://user-images.githubusercontent.com/98268472/151520337-851ddbda-a8ab-42b8-92e5-a26071900d7b.jpg)

**Comment by ghost** [comment]
Bence Yeni etkinlikler gelecektir.

**Reply by Yusufknk** [reply]
Evet 1 Şubat'tan sonra da etkinlikler olacaktır.
